UnifyEngine/FulldiveVRVideoProvidersUnifyEngineTests/Implementation/DocumentInterface.cs

```diff
@@ -33,7 +34,22 @@ public IEnumerable<string> GetImagesByCssQuery(string query)
         {
             return this._htmlDocument.DocumentNode.CssSelect(query)
                 .Where(x => x != null && x.HasAttributes)
-                .Select(x => x.GetAttributeValue("src"));
+                .Select(x => GetUrlImageArributeValue(x, _possibleImageUrlAttributeName);
+        }
+
+        private string GetUrlImageArributeValue(HtmlNode htmlNode, string[] altAttributeNames)
```

Comment:
Can be static

Reply:
37 :  .Select(x => GetUrlImageArributeValue(x, _possibleImageUrlAttributeName)); //mistake. Added ")" 
40 : private static string GetUrlImageArributeValue(HtmlNode htmlNode, string[] altAttributeNames)